CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@
 Changelog
 =========
 
+0.5.1 (unreleased)
+------------------
+
+**Bug fixes**
+
+- Public export of :func:`ndonnx.promote_nullable` is now working.
+
```

Comment:
```suggestion
0.6.0 (unreleased)
------------------

**Other changes**

- :func:`ndonnx.promote_nullable` is now publicly exported
```

I don't think this is a bug given it is explictly used in the tests through private APIs even in the tests. This would simply make it public.

Reply:
Side note - before doing this, I would like to enable user-defined data types to define their "nullable" counterpart.

Reply:
@adityagoel4512 One way or another there was a bug here: We were having `promote_nullable` in `__all__` but didn't import it into the scope.

Reply:
@adityagoel4512 judging by the `__init__` file alone I would think this is a bug, because the function is already listed in `__all__` but only the import was not working. But I'm ok with either way how we call this change.

Reply:
I never disagreed that the import did not work. Adding to the public API (which this does) is by definition a minor change however.